src/containers/FormikForm/TitleField.tsx

```diff
@@ -40,13 +39,18 @@ interface Props {
   hideToolbar?: boolean;
 }
 
-const StyledFormControl = styled(FormControl)`
-  margin-top: 2rem;
-  [data-title] {
-    font-size: 2.11111rem;
-    font-family: ${fonts.sans};
-  }
-`;
+const StyledRichTextEditor = styled(RichTextEditor, {
+  base: {
+    marginBlockStart: "medium",
+    textStyle: "heading.medium",
+  },
+});
+
+const StyledFieldHelper = styled(FieldHelper, {
+  base: {
+    color: "#8c8c00",
```

Comment:
hvorfor?

Reply:
Vi har ikke en farge på dette ennå. Hedvig må bestemme seg for en!

Reply:
Burde vi legge på en todo på at den skal byttes?